service/gcs/errors/errors.go

```diff
@@ -29,9 +35,9 @@ func (e *containerExistsError) Error() string {
 	return fmt.Sprintf("a container with the ID \"%s\" already exists", e.ID)
 }
 
-// NewContainerExistsError returns a *containerExistsError referring to the
+// NewContainerExistsError returns an error referring to the
```

Comment:
nit: this and some of the other doc comments below can be reduced to a single 80-character line now.

Reply:
Fixed.